docs/_articles/en/user-guide/development-models.md

```diff
@@ -49,6 +49,16 @@ After you provide an org alias, a browser window opens. Allow access, log in to
 
 After connecting to a sandbox, DE org, or Trailhead Playground, use the package.xml file to retrieve the metadata from your org. When you run SFDX: Create Project with Manifest command, a package.xml file is created. Add the various metadata types you want to retrieve to this file. To understand how to work with different subsets of metadata in `package.xml` file, see [Sample package.xml Manifest Files](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/manifest_samples.htm) in the _Metadata API Developer Guide_.
 
+### Manifest Builder
+Automatically generate a manifest file for a given a set of metadata components instead of editing the package.xml file manually: 
+1.  In the Explorer view, right-click to select the files you want to use to generate the manifest.
+2. Run **SFDX: Generate Manifest File**
+3. Enter a unique name (without an xml extension) for the manifest file. 
+4. A new file is created and added to the manifest folder.
```

Comment:
This isn't really a step (it's a step result).  Can you just indent it rather than assign it a step number?